worlds/pokemon_emerald/options.py

```diff
@@ -725,6 +725,25 @@ class FreeFlyLocation(Toggle):
     """
     display_name = "Free Fly Location"
 
+class FreeFlyBlacklist(OptionSet):
+    """
+    Disables specific locations as valid free fly locations.
+    Does nothing if Free Fly Location is set to false.
+    If all locations are disabled free fly location will be set to Littleroot Town.
```

Comment:
```suggestion
class FreeFlyBlacklist(OptionSet):
    """
    Disables specific locations as valid free fly locations.
    Does nothing if Free Fly Location is disabled.
```
I don't think this line is necessary for players. I think it's reasonable to assume that if there is no enabled free fly location, then the option effectively does nothing, which is what setting it to Littleroot does. In fact, it's ultimately identical on the mod side as well, because it just sets the city to "visited", which triggers for Littleroot as soon as you step out of the truck anyway. And the spoiler won't show free fly in the playthrough if it's not logically relevant.

Also, "set to false" is a very text editor way to word the second line. I think "disabled" might be more consistent with other options.